File: GTTS/INL.VettingService.Models/UpdateVettingBatch_Param.cs
using INL.Services.Models;
using System.Collections.Generic;

namespace INL.VettingService.Models
{
    public class UpdateVettingBatch_Param : BaseParam, IUpdateVettingBatch_Param
    {
        public UpdateVettingBatch_Param() : base() { }

        public long VettingBatchID { get; set; }
        public int? AssignedToAppUserID { get; set; }
        public int? ModifiedByAppUserID { get; set; }
        public string VettingBatchNotes { get; set; }
        public string VettingBatchStatus { get; set; }
        public int? VettingTypeID { get; set; }
        public string INKTrackingNumber { get; set; }
        public string LeahyTrackingNumber { get; set; }
    }
}

File: GTTS/GTTS.Web/ClientApp/src/app/models/INL.VettingService.Models/update-vetting-batch_param.ts


import { IUpdateVettingBatch_Param } from './iupdate-vetting-batch_param';

export class UpdateVettingBatch_Param implements IUpdateVettingBatch_Param {
  
	public VettingBatchID: number = 0;
	public AssignedToAppUserID?: number;
	public VettingBatchNotes: string = "";
	public VettingBatchStatus: string = "";
	public VettingTypeID?: number;
	public INKTrackingNumber: string = "";
	public LeahyTrackingNumber: string = "";
  
	public ErrorMessages: string[] = null;
}


